classes/PaymentModule.php

```diff
@@ -1104,6 +1104,7 @@ protected function createOrderCartRules(
             // THEN
             //  The voucher is cloned with a new value corresponding to the remainder
             $remainingValue = $cartRule->reduction_amount - $values[$cartRule->reduction_tax ? 'tax_incl' : 'tax_excl'];
+            $remainingValue = Tools::ps_round($remainingValue, _PS_PRICE_COMPUTE_PRECISION_);
```

Comment:
Thank you for your contribution @202-ecommerce  !

The _PS_PRICE_COMPUTE_PRECISION_ has been deprecated in 1.7.7 (see this PR #15084 for more informations)

You should be able to get the compute precision this way:

`Context::getContext()->getComputingPrecision()`

Reply:
I see `Context::getContext()->getComputingPrecision()` is used at line 1182 in this file